src/pages/AzurePreview.js

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { ButtonOutline, ButtonPrimary, Link } from 'src/components/common'
+import { ReactComponent as AzureLogo } from 'src/images/azure.svg'
+import planet from 'src/images/register-planet.svg'
+import { Ajax } from 'src/libs/ajax'
+import { signOut } from 'src/libs/auth'
+import { brands } from 'src/libs/brands'
+import colors from 'src/libs/colors'
+import { getConfig } from 'src/libs/config'
+import { reportErrorAndRethrow } from 'src/libs/error'
+import { terraLogoMaker } from 'src/libs/logos'
+import { useCancellation, useOnMount } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+
+
+const AzurePreview = () => {
+  // State
+  const [isAlphaAzureUser, setIsAlphaAzureUser] = useState(false)
+  const signal = useCancellation()
+
+  // Helpers
+  const loadAlphaAzureMember = reportErrorAndRethrow('Error loading azure alpha group membership')(async () => {
+    setIsAlphaAzureUser(await Ajax(signal).Groups.group(getConfig().alphaAzureGroup).isMember())
```

Comment:
The allow-list is controlled by the `alphaAzureGroup` Sam group.

There is a subtle chicken-and-egg problem here. The spec says this page needs to appear before the Register/ToS pages. But we can only check a user's group membership if they have already registered/accepted ToS. So it's actually impossible for an allow-listed Azure user to register in Terra UI.

A user can still register/accept ToS in Swagger, then use Terra UI as normal (this is indeed what I did for the above test user). Product/UX is okay with this as a first pass. As a follow-up we can investigate representing whether a user is allowed in B2C, which would sidestep this issue.

Reply:
Actually looking into moving the enabled check to the b2c side, will likely update this.

Reply:
This is good info that might be worth adding as a comment in the code.

Reply:
Yeah, and perhaps even a ticket. Will create/update.

Reply:
https://broadworkbench.atlassian.net/browse/TOAZ-301, will comment code too.

Reply:
Added comment